addons/phone_validation/views/res_partner_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
+        <record id="view_res_partner_filter" model="ir.ui.view">
+            <field name="name">res.partner.select.inherit.phone.validation</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base.view_res_partner_filter"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='email']" position="after">
```

Comment:
We could "replace" phone by phone_mobile_search (or hide phone and add phone mobile search after it). That way with base people search on classic phone, with SMS they have advanced search enabled, and everything is smooth.

Reply:
done.